test/e2e/auth/audit.go

```diff
@@ -35,6 +35,7 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	restclient "k8s.io/client-go/rest"
 	"k8s.io/kubernetes/test/e2e/framework"
+	e2elog "k8s.io/kubernetes/test/e2e/framework"
```

Comment:
this is importing k8s.io/kubernetes/test/e2e/framework under a different name, not using k8s.io/kubernetes/test/e2e/framework/log as I expected

Reply:
same comment applies to all places

Reply:
Ugh great catch, sorry about. Just a copy/paste error everywhere; my eyes never finished reading the line I guess. Patch coming.